bundles/ui/org.eclipse.smarthome.ui.icon/src/main/java/org/eclipse/smarthome/ui/icon/internal/IconServlet.java

```diff
@@ -41,6 +47,7 @@
  *
  * @author Kai Kreuzer - Initial contribution
  */
+@Component(configurationPolicy = ConfigurationPolicy.OPTIONAL)
```

Comment:
> If not specified, the OPTIONAL configuration policy is used. 

On no other places we set the configuration policy to optional explicit, why here?

Reply:
It was defined like that in the original service XML, see https://github.com/eclipse/smarthome/pull/5265#discussion_r197353799.

In case its optional I suggest addressing this in another PR for better traceability.

Reply:
@htreu What do you want to trace?
We migrate to the component annotations.
The annotations have default values, that does not need to be set explicitly.
We have requested to drop implicit values all the time (in this PR and all the others) if we found their usage in the review. Don't you remember on the comments about dropping the mandatory (1..1) reference and the static policy because it is the default value of the annotation? It does not matter, if this has been part of the XML before (e.g. https://github.com/eclipse/smarthome/pull/5265/files#diff-46cb4a3fb85d0b0057a3f9264fc8da74L18).

Reply:
Alright, thanks for the implicit explanation that `configurationPolicy = ConfigurationPolicy.OPTIONAL` is the default :-)
@lpapazow please drop it as requested.